src/orquestra/sdk/_base/_config.py

```diff
@@ -393,10 +393,7 @@ def update_config(
             left intact.
 
     Raises:
-        ValueError:
-            - if `config_name` was resolved to "local".
-            - if `config_name` couldn't be resolved automatically.
-        KeyError:
+        RuntimeConfigError:
```

Comment:
So far as I can see, ValueError and KeyError won't actually be raised here hence their removal. Please double check this in case I've missed something!